app/controllers/comments_controller.rb

```diff
@@ -47,19 +48,22 @@ def destroy
       @delete.destroy
       redirect_to :back
     else
-      flash[:alert] = "You can't delete this comment!"
+      flash[:alert] = 'You can\'t delete this comment!'
       redirect_to :back
     end
   end
 
   private
-
     def logged_in
-      redirect_to root_url unless !current_user.nil?
+      redirect_to root_url if current_user.nil?
     end
 
     def comment_params
-      params.require(:comment).permit(:polycomment_id,:polycomment_type,:issue,:body)
+      params.require(:comment).permit(
+        :polycomment_id,
+        :polycomment_type,
+        :issue,
+        :body
```

Comment:
awesome, I find this way of doing it so much more easier to work with.

I remember once just cleaning up as much occurrence of parameter formatting as I could find, without which, my eyes would just.. refuse to work :D